spark-jobs/src/test/resources/interpreter/mappingCases/array1Schema.txt

```diff
@@ -39,4 +39,4 @@ root
  |    |    |-- skey2: string (nullable = true)
  |    |    |-- conformedNum3: struct (nullable = true)
  |    |    |    |-- id: long (nullable = true)
- |    |    |    |-- name: string (nullable = true)
\ No newline at end of file
+ |    |    |    |-- name: string (nullable = true)
```

Comment:
Ditto with schema?

Reply:
Why with schema? The differences observed on schemas (nullability changes) would be the same if parsed from text or if provided in code or otherwise.